test/GithubApi.Put.test.js

```diff
@@ -0,0 +1,32 @@
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+
+const path = 'https://api.github.com/user/following';
+let response;
+
+const auth = axios.create({
+  baseURL: path,
+  headers: { Authorization: `token ${process.env.ACCESS_TOKEN}` }
+});
+
+describe('Consuming PUT Service GitHub API', () => {
+  it('Following User', async () => {
+    response = await auth.put(`${path}/aperdomob`);
+    expect(response.status).to.equal(StatusCodes.NO_CONTENT);
+    expect(response.data).to.be.eql('');
+  });
+
+  it('Check Follow User', async () => {
+    response = await auth.get(`${path}`);
+    const userFollow = response.data.find((e) => e.login === 'aperdomob');
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(userFollow.login).to.equal('aperdomob');
+  });
```

Comment:
Good job! as a suggestion, there is a rule that tests should be independent. That means that what is inside an 'it' should not depend on the execution of the other 'its'. To solve that problem, try to use the responses that are needed on many tests ('it') outside of them, inside a `before` step to be sure that it will be executed before any of the tests. I suggest to refactor to something like:

```suggestion
describe('Consuming PUT Service GitHub API', () => {
  before(async => {
    response = await auth.put(`${path}/aperdomob`);
  }
  it('Following User', async () => {
    expect(response.status).to.equal(StatusCodes.NO_CONTENT);
    expect(response.data).to.be.eql('');
  });
  describe ('check following correct users', () => {
    before(async => {
          response = await auth.get(`${path}`);
    }
    it('Check Follow User', async () => {
      const userFollow = response.data.find((e) => e.login === 'aperdomob');
      expect(response.status).to.equal(StatusCodes.OK);
      expect(userFollow.login).to.equal('aperdomob');
  });
  }
```

And same logic for next section :)

Reply:
Here you can read about Hooks from mocha: https://mochajs.org/#hooks